### PR TITLE
Slider bug fix

### PR DESCRIPTION
**Pull request by yuda1124**
handle slider index when image is clicked and report error fix

### CODE REVIEW

File: shared/components/detailView.js [hunk 1]
```diff
@@ -399,6 +397,7 @@ export default class DetailView extends Component {
             loop = {false}>
               {pages}
           </Swiper>
+          {this.renderModal()}
```
Comment (ochanje210) [inline shared/components/detailView.js]: why did u move it down?

Comment (yuda1124) [inline shared/components/detailView.js]: Because that looks better!

Comment (ochanje210) [inline shared/components/detailView.js]: no problem! :)

**Comment by ochanje210** [comment]
lgtm!!

**Pull request merged**